core/src/test/scala/unit/kafka/coordinator/group/GroupMetadataTest.scala

```diff
@@ -609,7 +609,7 @@ class GroupMetadataTest {
   @Test
   def testReplaceGroupInstanceWithNonExistingMember(): Unit = {
     val newMemberId = "newMemberId"
-    assertThrows(classOf[IllegalArgumentException], () => group.replaceStaticMember(groupInstanceId, memberId, newMemberId))
+    assertThrows(classOf[IllegalArgumentException], () => group.replaceStaticMember(groupInstanceId, memberId, newMemberId, clientId, clientHost))
   }
 
   @Test
```

Comment:
If you agree to my comment [here](https://github.com/apache/kafka/pull/14666/files#diff-1c062c229499ca8f215e0088c8ab0dfee768025cd3b59042132fddf70aef4c0cR1342), then can we add a test which validates that the clientId and clientHost gets reverted?

Reply:
Sure